### ISSUE TITLE
NYISO Interconnection Queue bug in latest release

### ISSUE THREAD

**Issue by bendnorman** [kmax12/gridstatus#142]
When I run:

```python
import gridstatus
nyiso = gridstatus.NYISO()
nyiso_queue = nyiso.get_interconnection_queue()
```
using gridstatus `0.17.0` I get this error:

```
ValueError                                Traceback (most recent call last)
Cell In[310], line 3
      1 import gridstatus
      2 nyiso = gridstatus.NYISO()
----> 3 nyiso_queue = nyiso.get_interconnection_queue()

File ~/.local/lib/python3.9/site-packages/gridstatus/nyiso.py:279, in NYISO.get_interconnection_queue(self, verbose)
    277 completed.insert(17, "Proposed Initial-Sync", None)
    278 completed["Status"] = InterconnectionQueueStatus.COMPLETED.value
--> 279 completed.columns = active.columns
    281 # the spreadsheet doesnt have a date, so make it null
    282 completed["Proposed  In-Service"] = None

File ~/.local/lib/python3.9/site-packages/pandas/core/generic.py:5500, in NDFrame.__setattr__(self, name, value)
   5498 try:
   5499     object.__getattribute__(self, name)
-> 5500     return object.__setattr__(self, name, value)
   5501 except AttributeError:
   5502     pass

File ~/.local/lib/python3.9/site-packages/pandas/_libs/properties.pyx:70, in pandas._libs.properties.AxisProperty.__set__()

File ~/.local/lib/python3.9/site-packages/pandas/core/generic.py:766, in NDFrame._set_axis(self, axis, labels)
    764 def _set_axis(self, axis: int, labels: Index) -> None:
    765     labels = ensure_index(labels)
--> 766     self._mgr.set_axis(axis, labels)
    767     self._clear_item_cache()

File ~/.local/lib/python3.9/site-packages/pandas/core/internals/managers.py:216, in BaseBlockManager.set_axis(self, axis, new_labels)
    214 def set_axis(self, axis: int, new_labels: Index) -> None:
    215     # Caller is responsible for ensuring we have an Index object.
--> 216     self._validate_set_axis(axis, new_labels)
    217     self.axes[axis] = new_labels

File ~/.local/lib/python3.9/site-packages/pandas/core/internals/base.py:57, in DataManager._validate_set_axis(self, axis, new_labels)
     54     pass
     56 elif new_len != old_len:
---> 57     raise ValueError(
     58         f"Length mismatch: Expected axis has {old_len} elements, new "
     59         f"values have {new_len} elements"
     60     )

ValueError: Length mismatch: Expected axis has 20 elements, new values have 21 elements
```

However, when I install gridstatus from the main branch I don't receive this error. Is it possible to do a release soon so the patch to this bug is accessible in the pypi package? Thank you!

**Comment by kmax12** [comment]
0.18.0 release is out: https://pypi.org/project/gridstatus/

closing for now. let me know if you hit any other issues

**Comment by bendnorman** [comment]
Wow, thank you for doing a release so quickly! `0.18.0` is working for me. 

We are considering using gridstatus instead of this [manually curated set of projects](https://emp.lbl.gov/generation-storage-and-hybrid-capacity) created by LBNL, which is released once a year. I did a quick comparison and the number of projects and total capacity seem roughly the same. Will keep you up to date on any quicks we find!

**Comment by bendnorman** [comment]
Hey @kmax12! I think this issue has popped up again in 0.18.0:

```
ValueError                                Traceback (most recent call last)
Cell In[12], line 1
----> 1 gs.NYISO().get_interconnection_queue()

File ~/.local/lib/python3.9/site-packages/gridstatus/nyiso.py:293, in NYISO.get_interconnection_queue(self, verbose)
    291 completed.insert(18, "Proposed Initial-Sync Date", None)
    292 completed["Status"] = InterconnectionQueueStatus.COMPLETED.value
--> 293 completed.columns = active.columns
    295 # the spreadsheet doesnt have a date, so make it null
    296 completed["Proposed  In-Service"] = None

File ~/.local/lib/python3.9/site-packages/pandas/core/generic.py:5500, in NDFrame.__setattr__(self, name, value)
   5498 try:
   5499     object.__getattribute__(self, name)
-> 5500     return object.__setattr__(self, name, value)
   5501 except AttributeError:
   5502     pass

File ~/.local/lib/python3.9/site-packages/pandas/_libs/properties.pyx:70, in pandas._libs.properties.AxisProperty.__set__()

File ~/.local/lib/python3.9/site-packages/pandas/core/generic.py:766, in NDFrame._set_axis(self, axis, labels)
    764 def _set_axis(self, axis: int, labels: Index) -> None:
    765     labels = ensure_index(labels)
--> 766     self._mgr.set_axis(axis, labels)
    767     self._clear_item_cache()

File ~/.local/lib/python3.9/site-packages/pandas/core/internals/managers.py:216, in BaseBlockManager.set_axis(self, axis, new_labels)
    214 def set_axis(self, axis: int, new_labels: Index) -> None:
    215     # Caller is responsible for ensuring we have an Index object.
--> 216     self._validate_set_axis(axis, new_labels)
    217     self.axes[axis] = new_labels

File ~/.local/lib/python3.9/site-packages/pandas/core/internals/base.py:57, in DataManager._validate_set_axis(self, axis, new_labels)
     54     pass
     56 elif new_len != old_len:
---> 57     raise ValueError(
     58         f"Length mismatch: Expected axis has {old_len} elements, new "
     59         f"values have {new_len} elements"
     60     )

ValueError: Length mismatch: Expected axis has 20 elements, new values have 21 elements
```

**Comment by kmax12** [comment]
I had to fix it again last week in #161. 

if you upgrade to `0.19.0` it should work
